src/Sylius/Bundle/AdminBundle/Resources/views/Taxon/_treeWithButtons.html.twig

```diff
@@ -34,10 +34,10 @@
 
                                     <div class="divider"></div>
 
-                                    <div class="item sylius-taxon-move-up" data-url="{{ path('sylius_admin_ajax_taxon_move', { id: taxon.id }) }}" data-id="{{ taxon.id }}" data-position="{{ taxon.position }}">
+                                    <div class="item sylius-taxon-move-up" data-url="{{ path('sylius_admin_ajax_taxon_move_up', { id: taxon.id }) }}">
```

Comment:
It might be worth mentioning in UPGRADE as functionality will break for anyone with overridden template

Reply:
Good idea, added.